terraform/envs/dev/variables.tf

```diff
@@ -109,3 +109,18 @@ variable "dcp_auth0_audience" {
 variable "gcp_service_acct_creds" {
   type = "string"
 }
+
+// AWS RDS Cluster instance
+
+variable "aws_rds_cluster_instance_class" {
```

Comment:
I think we should git-commit the value we use here. That way devs can review changes to instance sizes, postgres versions, etc. in PRs (the big plus of infrastructure-as-code). But I'm not sure if the terraform deploy scripts for upload-service are set up in that way, from what I remember when working on them

Reply:
all tfvars are currently being sourced from s3

Reply:
Should be possible to do but maybe that's also intentional. The buckets are versioned and can be reviewed, i guess it's fine for now.

 The changes are:
for dev and staging:
```
// AWS RDS Cluster Instance
aws_rds_cluster_instance_class="db.t3.medium"
aws_rds_cluster_instance_engine_version="10.11"
aws_rds_db_cluster_parameter_group_name="default.aurora-postgresql10"
```
for prod:
```
// AWS RDS Cluster Instance
aws_rds_cluster_instance_class="db.r4.large"
aws_rds_cluster_instance_engine_version="9.6.12"
aws_rds_db_cluster_parameter_group_name="default.aurora-postgresql9.6"
```